beekeeper-metadata-cleanup/src/main/java/com/expediagroup/beekeeper/metadata/cleanup/cleaner/MetadataCleanup.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (C) 2019-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.metadata.cleanup.cleaner;
+
+import java.time.LocalDateTime;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+import com.expediagroup.beekeeper.core.model.HousekeepingMetadata;
+
+public interface MetadataCleanup {
```

Comment:
```suggestion
public interface MetadataCleaner {
```
thoughts on this rename?

Reply:
We're not sure on the naming we want to go for. Initially Vedant put that, but theres already a `MetadataCleaner` interface inside the beekeeper-cleanup module, which is implemented by the `HiveMetadataCleaner`, so went for this instead

Reply:
I've removed the class that was previously `MetadataHandler` as it wasnt doing a whole lot (just processing pages), so now this interface and its impl have been changed to `MetadataHandler` and `ExpiredMetadataHandler` 👍